boilerplates/ts-rest/files/ts-rest/client.ts

```diff
@@ -0,0 +1,9 @@
+import { initClient } from "@ts-rest/core";
+import { contract } from "./contract";
+
+const port = import.meta.env.PORT ? parseInt(import.meta.env.PORT, 10) : 3000;
+
+export const client = initClient(contract, {
+  baseUrl: `http://localhost:${port}`,
```

Comment:
If it's not mandatory, prefer not using `baseUrl` at all. Otherwise, add a comment above it: `TODO: properly retrieve host`

Reply:
And `baseUrl: '/'` doesn't work either? If not, the best thing would probably be to retrieve necessary information from [`location`](https://developer.mozilla.org/en-US/docs/Web/API/Location)

Reply:
`baseUrl: "/"` doesn't work, the result will be `http://api/todo/create`.

What about this: 
```ts
const port = typeof location !== "undefined" ? location.port : "3000"
```

Since `location` will be undefined when `ssr`

Reply:
It seems to work with empty string `baseUrl: ""`

Reply:
Indeed that works, I've pushed a change :+1: